gamestonk_terminal/portfolio/portfolio_helper.py

```diff
@@ -71,12 +73,16 @@ def get_fraction(numerator: float, denominator: float) -> str:
         The numerator
     denominator : float
         The denominator
+    allow_negative_denominator : bool
+        If false, a negative denominator will return "NA"
 
     Returns
     ----------
-    text : str
+    str
         A fraction as a string
     """
+    if allow_negative_denominator and denominator != 0:
```

Comment:
I think this is overcomplicating, why not letting as it was and calling the function
```
def get_fraction_with_positive_denominator(
     numerator: float, denominator: float
 ) -> str:
     """Turn into string the result of the numerator divided by the denominator, in which the denominator must be a positive value
     ....
```

Reply:
I only like this way better in case I ever add ratios that can have negative denominators. If you want me to change it I can.

Reply:
I think this function is a bit overkill as is, IMHO because you can do:
```
(numerator/denominator):.2f if denominator > 0 else "N/A"
```
And is more readable than having the function itself

Reply:
I like that better.